pkg/apis/core/types.go

```diff
@@ -2150,6 +2150,12 @@ type ExecAction struct {
 	Command []string
 }
 
+// SleepAction describes a "sleep" action.
+type SleepAction struct {
+	// Seconds is the number of seconds to sleep.
+	Seconds int64
```

Comment:
There is an ongoing KEP about using sub-seconds for probes in https://github.com/kubernetes/enhancements/issues/3066.

I wondered if we could use milliseconds here. Better to change it after we get confirmed from API-approvers.

Reply:
sleep milliseconds has any use case?